modules/apps/portal/portal-company-log/src/main/java/com/liferay/portal/company/log/internal/servlet/CompanyLogServlet.java

```diff
@@ -159,6 +168,35 @@ private void _downloadLogFile(
 		}
 	}
 
+	private File _getlogFilesDir(long companyId) throws Exception {
+		Logger rootLogger = (Logger)LogManager.getRootLogger();
```

Comment:
@yuhai we want to avoid Log4J API usages whenever possible outside Log4J related module (core, petra-log4j, module-log4j-extender..)

For this particular need, could you try to register an dummy OSGi service with the company id and log file path as properties when creating an appender for a company? This way you can just track it here without using Log4J APIs.

cc @tinatian

Reply:
@dantewang, I will try to do this.